pages/use-solid/apps.md

```diff
@@ -144,10 +144,11 @@ redirect_from:
 | [SPARQL Fiddle](https://jeff-zucker.github.io/sparql-fiddle/)                | Online fiddle to run SPARQL against  Pods. <br>- [Source code](https://github.com/jeff-zucker/sparql-fiddle) [MIT License Copyright (c) 2018](https://github.com/jeff-zucker/sparql-fiddle/blob/master/LICENSE) [Jeff Zucker](https://github.com/jeff-zucker) |
 | [Assemblage](https://gitlab.com/omd_public/assemblage) | A composable data browser aimed to provide a great user experience for Solid POD services, as well as other RDF sources, both for desktop and mobile web browsers. [Source code](https://gitlab.com/omd_public/assemblage) and [demo video](https://youtu.be/hYFEBAdkzmc). |
 | [Warp](https://linkeddata.github.io/warp/)                                   | File browser. <br>- [Source code](https://github.com/linkeddata/warp) [MIT License Copyright (c) 2014](https://github.com/linkeddata/warp/blob/gh-pages/LICENSE) [Andrei Samba](https://github.com/deiu) |
+| [Penny](https://penny.vincenttunru.com/)                                   | File browser. <br>- [Source code](https://github.com/linkeddata/warp) [MIT License Copyright (c) 2014](https://github.com/linkeddata/warp/blob/gh-pages/LICENSE) [Vincent Turu](https://gitlab.com/Vinnl) |
```

Comment:
Hey Nick -- 
this row and the below row seems to have gotten somehow entangled.

This Penny row seems to link to the Warp application  whereas below, in the Historical Solid Apps section, you seem to have clobbered Mark Book source and license link with Vincent's